tests/functional-tests/README.md

```diff
@@ -0,0 +1,77 @@
+# IBM Spectrum Scale CSI - Automation
+
+The IBM Spectrum Scale Container Storage Interface (CSI) project enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage.
```

Comment:
Do we need  this line here?  it seems this should be part of the higher level README ..